docs/resources/zk.md

```diff
@@ -0,0 +1,94 @@
+---
+page:
+  headHtml: |
+    <snippet var="js.highlightjs" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/languages/ini.min.js"></script>
+---
+
+# zk
+
+[`zk`](https://github.com/mickael-menu/zk) is a command-line tool for working
+with Zettelkasten projects. It focuses on being easy for automation and is
+generic in the sense that it does not lock in to only supporting a certain note
+taking tool, which makes it useful for [Obsidian](https://obsidian.md/),
+[Neuron](https://neuron.zettel.page/), and Emanote notes collections.
+
+## Configuring repo
+
+```toml
+[root]
+foo = "bar"
+```
```

Comment:
This TOML config is placed here without any context. What is the reader supposed to do with this? Why does it say "foo"? What is the meaning of this config?

Reply:
Lol I accidentally committed my test code from #320. Nice catch, will remove!

Reply:
Removed